src/Fixer/ConstantNotation/NativeConstantInvocationFixer.php

```diff
@@ -0,0 +1,169 @@
+    {
+        $constantNames = $this->getConstantNames();
+
+        $indexes = [];
+
+        for ($index = 0, $count = $tokens->count(); $index < $count; ++$index) {
+            $token = $tokens[$index];
+
+            $tokenContent = $token->getContent();
+
+            // test if we are at a constant call
+            if (!$token->isGivenKind(T_STRING)) {
+                continue;
+            }
+
+            $next = $tokens->getNextMeaningfulToken($index);
+            if ($tokens[$next]->equals('(')) {
+                continue;
+            }
+
+            $constantNamePrefix = $tokens->getPrevMeaningfulToken($index);
+            if ($tokens[$constantNamePrefix]->isGivenKind([CT::T_USE_TRAIT, T_CLASS, T_CONST, T_DOUBLE_COLON, T_EXTENDS, T_FUNCTION, T_IMPLEMENTS, T_INTERFACE, T_NAMESPACE, T_NEW, T_NS_SEPARATOR, T_OBJECT_OPERATOR, T_TRAIT, T_USE])) {
+                continue;
+            }
+
+            if (!\in_array($tokenContent, $constantNames, true)) {
+                continue;
+            }
+
+            $indexes[] = $index;
+        }
+
+        $indexes = \array_reverse($indexes);
+        foreach ($indexes as $index) {
+            $tokens->insertAt($index, new Token([T_NS_SEPARATOR, '\\']));
+        }
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function createConfigurationDefinition()
+    {
+        $constantChecker = function ($value) {
+            foreach ($value as $constantName) {
+                if (!\is_string($constantName) || '' === \trim($constantName) || \trim($constantName) !== $constantName) {
+                    throw new InvalidOptionsException(\sprintf(
+                        'Each element must be a non-empty, trimmed string, got "%s" instead.',
+                        \is_object($constantName) ? \get_class($constantName) : \gettype($constantName)
+                    ));
+                }
+            }
+
+            return true;
+        };
+
+        return new FixerConfigurationResolver([
+            (new FixerOptionBuilder('include', 'List of constants to fix. Default is to fix all constants returned by `get_defined_constants`.'))
+                ->setAllowedTypes(['array'])
+                ->setAllowedValues([$constantChecker])
+                ->setDefault([])
+                ->getOption(),
+            (new FixerOptionBuilder('exclude', 'List of constants to ignore.'))
+                ->setAllowedTypes(['array'])
+                ->setAllowedValues([$constantChecker])
+                ->setDefault([])
+                ->getOption(),
+        ]);
+    }
+
+    /**
+     * @return string[]
+     */
+    private function getConstantNames()
+    {
+        $definedConstants = $this->configuration['include'];
+        if (empty($definedConstants)) {
+            $definedConstants = \get_defined_constants();
+            unset($definedConstants['NULL'], $definedConstants['FALSE'], $definedConstants['TRUE']);
```

Comment:
for a phar release, I would like to `\` all native constants, including null/false/true. With which configuration I could achieve it ?